test/uk/gov/hmrc/plasticpackagingtax/registration/views/group/ContactDetailsNameViewSpec.scala

```diff
@@ -111,7 +109,7 @@ class ContactDetailsNameViewSpec extends UnitViewSpec with Matchers {
 
       view must haveGovukGlobalErrorSummary
 
-      view must haveGovukFieldError("firstName", "Enter first name")
+      view must haveGovukFieldError("firstName", "Enter the main contact’s first name")
```

Comment:
can we use the messagesApi to get this

Reply:
If there a issue in the messages file i suppose these tests wouldn't pick up the difference

Reply:
Yeah... lots of this sort of thing in the code base.... If the exact wording is important enough that it's needs to be tested then an explicit value. Otherwise does it -really- need a test at all?